hw/ip/edn/data/edn_testplan.hjson

```diff
@@ -97,10 +97,9 @@ covergroups: [
             Covers that all fatal errors, all fifo errors and all error codes of edn
             have been tested.
             Individual config settings that will be covered include:
-            - which_fatal_err (0 to 4), 5 possible fatal errors
-            - which_fifo_write_err (0 to 1), 2 differnt fifos
-            - which_fifo_read_err (0 to 1), 2 different fifos
-            - which_fifo_state_err (0 to 1), 2 different fifos
+            - which_fatal_err (0 to 4), 5 possible fatal errors including rescmd, gencmd fifo
+              error, ack_sm_err, main_sm_err and cntr_err
+            - which_fifo (0 to 1), 2 differnt fifos, rescmd fifo and gencmd fifo
```

Comment:
Do you mean to collapse which_fifo? But which_fifo and which_fifo_err covers different testing aspects, which_fifo_err specifies which type of fifo errors, either write error, read error or state error, but which_fifo specifies which fifo will be forced to have write/read/state error.

Reply:
ah, sorry to be unclear.  This wasn't a suggestion.  I saw that you had `which_fifo_write_err` and `which_fifo_read_err`, and I think it's good that you collapsed them because I see that you have `which_fifo_err` below.  No action required, just commenting in agreement with the change.

Reply:
okay, I see. Thanks!